packages/eslint-plugin/tests/rules/no-unnecessary-type-assertion.test.ts

```diff
@@ -6,13 +6,19 @@ import rule from '../../src/rules/no-unnecessary-type-assertion';
 const rootDir = path.resolve(__dirname, '../fixtures/');
 const ruleTester = new RuleTester({
   parserOptions: {
+    EXPERIMENTAL_useProjectService: false,
```

Comment:
Without this, `CI / Run Unit Tests with Experimental TSServer (eslint-plugin) (pull_request)` failed